versions.tf

```diff
@@ -1,7 +1,10 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = ">= 0.13"
```

Comment:
May I ask why >= instead of ~>?

Reply:
tbh I would've put `~>` but `terraform 0.13upgrade` auto-suggested this so I rolled with it. Since it's a module template (and therefore a starting point) I left it that way.